tests/std/tests/P0896R4_ranges_alg_swap_ranges/test.cpp

```diff
@@ -95,6 +95,8 @@ constexpr void run_tests() {
     instantiator::call<test_range<bidi, ProxyRef::yes>, test_range<input, ProxyRef::yes>>();
     instantiator::call<test_range<random, ProxyRef::yes>, test_range<input, ProxyRef::yes>>();
     instantiator::call<test_range<contiguous, ProxyRef::no>, test_range<input, ProxyRef::yes>>();
+
+    instantiator::call<test_range<contiguous, ProxyRef::no>, test_range<contiguous, ProxyRef::no>>();

```

Comment:
I am slightly paranoid about the test ranges not triggering the optimization. Have you verified that it indeed uses the optimization at runtime?

Reply:
Yes